### DISCUSSION TITLE
Trying to resolve "unable to get local issuer certificate"

### DISCUSSION THREAD

**Discussion by jesse-beckton-alight** [vercel/turborepo#7940] [Help]
### Summary

Within my Orgs network I have been trying to resolve local cert issues with npm / npx. When I use the "npx create-turbo" command it fails. None of the fixes I have found to date work. My question is how does the cli access the templates? does it use an https:// or git:// or... ? I have been working with my IT support group for almost 2 weeks trying to resolve this issue with no luck, but maybe if I knew what was going on under the covers I could better assist my IT dept to come up with a fix.

>>> Unable to download template from Github
>>> unable to get local issuer certificate

Thanks.

**Comment by anthonyshew** [comment]
Are you using GitHub Enterprise by chance?

If so, it sounds like this Issue may be what you're running into: https://github.com/vercel/turbo/issues/5945

**Reply by jesse-beckton-alight** [reply]
Yes we are. My org uses Github Ent. Thanks for the tip.

**Reply by jesse-beckton-alight** [reply]
not the same issue, I am just trying to use the templates Turbo already provides, they are not private repos.
